### PR TITLE
Introduces sql operations to deal with database changes

### PR DESCRIPTION
**Pull request by jorgeantonio21**
Introduces interface to deal with database changes operations (delete, insert, update). It also adds flexible interface to deal with both Diesel Rust and PostgreSql types, in a way to reduce any possible Sql injections. This PR addresses issue #1.

Already has some unit tests, but currently unable to run it on my local machine (clang issues with mac m1 x86)

### CODE REVIEW

File: README.com [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Substreams Sink PostgreSQL
```
Comment (benjaminbollen) [inline README.com]: this file is outdated, could you remove?

File: eureka-sink-postgres/db/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[package]
+name = "substreams-postgres-sync"
```
Comment (benjaminbollen) [inline eureka-sink-postgres/db/Cargo.toml]: this crate name can be made more accurate

File: eureka-sink-postgres/sinker/Cargo.toml [hunk 3]
```diff
@@ -7,12 +7,13 @@ edition = "2021"
 
 
 [dependencies]
-prost = { version = "0.10.1" }
-prost-types = "0.10.1"
-tonic = "0.7.1"
+prost = { version = "^0.11.0" }
```
Comment (benjaminbollen) [inline eureka-sink-postgres/sinker/Cargo.toml]: all of `/sinker` we can snapshot onto a branch and come back to later; for now I won't review in detail

**Review by benjaminbollen — APPROVED** [review]
A README.md in `/db` would be good. FIPF

File: README.com (hunk 1 above)
Comment (jorgeantonio21) [reply README.com]: Done

File: eureka-sink-postgres/sinker/Cargo.toml (hunk 3 above)
Comment (jorgeantonio21) [reply eureka-sink-postgres/sinker/Cargo.toml]: Agreed, I will remove it in this PR.

**Comment by jorgeantonio21** [comment]
Address the comments

**Review by benjaminbollen — APPROVED** [review]
FIPF

**Pull request merged**